tests/metrics/storage/fio-k8s/fio-test-ci.sh

```diff
@@ -79,8 +78,6 @@ EOF
 	metrics_json_add_array_element "$json"
 	metrics_json_end_array "Results"
 	metrics_json_save
-
-	check_processes
```

Comment:
Is this check no longer required? Can you mention in the commit message why this call is no longer required?